api/resources_portal/views/material_request.py

```diff
@@ -170,9 +175,9 @@ def update(self, request, *args, **kwargs):
                 material_request.status = serializer.validated_data["status"]
 
             if "executed_mta_attachment" in request.data:
-                material_request.executed_mta_attachment = serializer.validated_data[
-                    "executed_mta_attachment"
-                ]
+                executed_mta = serializer.validated_data["executed_mta_attachment"]
+                material_request.executed_mta_attachment = executed_mta
+                assign_perm("view_attachment", material_request.assigned_to, executed_mta)
```

Comment:
Same answer to https://github.com/AlexsLemonade/resources-portal/pull/211/files#r442897370 applies here as well.